site/docs/resources/faq.md

```diff
@@ -26,6 +27,41 @@ No, it's not.
    As always, TypeScript will auto-complete the parameter names for you.
 3. Perhaps double-check the method signature for [actions](../guide/context#available-actions) on `ctx` [here](https://deno.land/x/grammy/mod.ts?s=Context#Methods), or for API methods (`ctx.api`, `bot.api`) [here](https://deno.land/x/grammy/mod.ts?s=Api#Methods).
 
+## How Can I Access the Chat History?
+
+You can't.
+
+Telegram does not store the messages for your bot.
+
+Instead, you need to wait for new messages/channel posts to arrive, and store the messages in your database.
+You can then load the chat history from your database.
+
+This is what [conversations](../plugins/conversations) do internally for the relevant part of the message history.
+
+## How Can I Handle Albums?
+
+You can't ... at least not in the way you think.
```

Comment:
Does the build step turn this into …? Else we should fix it

Reply:
It does, nvm

Reply:
I don’t think so. The typographer plugin does that when rendering.

Reply:
That is part of the build step, it's what I was referring to :)